### PR TITLE
Turn off backplate LED when program stops

### PR DESCRIPTION
**Pull request by mathoudebine**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -58,6 +58,9 @@ def clean_stop(tray_icon=None):
         # Turn screen off before stopping
         display.lcd.ScreenOff()
 
+        # Turn backplate LED off for supported devices
+        display.lcd.SetBackplateLedColor(led_color=(0, 0, 0))
```
Comment (amiltonjr) [inline main.py:62]: Is there a way to call this when the task/process is killed, as [shown here](https://stackoverflow.com/a/72592788)?

Comment (Rollbacke) [reply main.py:62]: Hi @amiltonjr,
Mathoudebine talks about it here: #132

**Comment by Rollbacke** [comment]
Hi @mathoudebine,
Tested today, it work.
Thanks :D

File: main.py (hunk 1 above)
Comment (mathoudebine) [reply main.py:62]: I will also implement the atexit callback from your link in #132

**Pull request merged**